cmake/options.cmake

```diff
@@ -4,7 +4,7 @@
 #########################################################################
```

Comment:
Please add this below copy right information after line number 2

Copyright (C) 2022 - 2024 Advanced Micro Devices, Inc. All Rights Reserved.

You can look into below file for reference
example/sys/linux/dfx-mgr-client.c

Reply:
Just updated the PR to include correct copyright information.